baseballAPI/ApiModels/BattingStatsCalculator.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+
+namespace BaseballAPI.ApiModels
+{
+    public class BattingStatsCalculator : IBattingStatsCalculator
+    {
+        public BattingStats CalculateStats(BattingStats batting)
+        {
+            batting.Singles = CalculateSingles(batting);
+
+            batting.Avg = CalculateAvg(batting);
+
+            batting.Obp = CalculateObp(batting);
+
+            batting.Slg = CalculateSlg(batting);
+
+            batting.Ops = CalculateOps(batting);
+
+            return batting;
+        }
+
+        private int CalculateSingles(BattingStats batting)
+        {
+            return batting.H - (batting.X2b + batting.X3b + batting.Hr);
+        }
+        private double CalculateAvg(BattingStats batting)
+        {
+            if(batting.Ab == 0)
+            {
+                return 0;
+            }
+
+            double avg = batting.H / batting.Ab;
+            
+            return Math.Round(avg, 4);
+        }
+
+        private double CalculateObp(BattingStats batting)
+        {
+            if (batting.Ab == 0)
+            {
+                return 0;
+            }
+
+            double obp = (batting.H + batting.Bb + batting.Ibb) / batting.Ab;
+
+            return Math.Round(obp, 4);
+        }
+
+        private double CalculateSlg(BattingStats batting)
+        {
+            if(batting.Ab == 0)
```

Comment:
This, `if divide by zero just return 0` paradigm might be worth pulling out to a helper function.
If this is a convention you want a lot, commonly this is pulled into something like:
``` C#
public static double SafeDivide(double numerator, double denominator) {
    return denominator != 0 ? numinator / denominator : 0;
}
```
(also seen names like `DivideOrDefault`, `DivideOrZero`, etc...)